src/settings/TargetProfileDetail.js

```diff
@@ -25,17 +38,71 @@ class TargetProfileDetail extends React.Component {
       path: 'identifier-types?query=id=!{initialValues.externalIdentifierType}',
       records: 'identifierTypes',
     },
+    jobProfiles: {
+      type: 'okapi',
+      records: 'jobProfiles',
+      path: 'data-import-profiles/jobProfiles?limit=5000&query=dataType==("MARC") sortBy name',
+    },
+  });
+
+  getJobProfilesContent = (profileIds, defaultProfileId) => {
+    const { resources: { jobProfiles } } = this.props;
+
+    const content = [...profileIds];
+    const indexOfDefault = content.indexOf(defaultProfileId);
+
+    if (indexOfDefault > -1) {
+      content.splice(indexOfDefault, 1);
+      content.unshift(defaultProfileId);
+    }
+
+    return content.map(id => ({
+      id,
+      name: jobProfiles?.records.find(jobProfile => jobProfile.id === id)?.name,
+    }));
+  }
+
+  getJobProfilesFormatter = defaultProfileId => ({
```

Comment:
Could you please investigate possible of usage const instead of hard code? example below
```
const COLUMNS_NAME = {
  ID: 'id',
  IS_DEFAULT: 'isDefault',
};
const jobProfilesVisibleColumns = [COLUMNS_NAME.ID, COLUMNS_NAME.IS_DEFAULT];
from id: ({ name, id }) to [COLUMNS_NAME.ID]: ({ name, id })
from id: <FormattedMessage id="ui-inventory.importCreateJobProfileId" />, to [COLUMNS_NAME.ID]: <FormattedMessage id="ui-inventory.importCreateJobProfileId" />,
```

Reply:
done